modules/order/src/Entity/OrderInterface.php

```diff
@@ -222,4 +223,23 @@ public function getBillingProfileId();
    */
   public function setBillingProfileId($billingProfileId);
 
+  /**
+   * Adds an adjustment to the order.
+   *
+   * @param \Drupal\commerce_order\Adjustment $adjustment
+   *    The adjustment.
+   *
+   * @return $this
+   *
+   * @todo: How do we remove or clear them?
```

Comment:
We never talked about this. We'll need some sort of "Remove" button in the UI. But how about programmatically?